math/src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable  */
+import React, { Component } from 'react';
+
+import "./Calculator.css";
+
+class Calculator extends React.Component {
```

Comment:
- [ ] **Please make sure that the calculator component follows the provided design guideline.**

Being able to follow a design guideline is a very important skill for any developer. Although your design appears to be exceptional, the motive of providing a design guideline is to make sure that you are well equipped to transform any design into a functional solution. Please make sure that the design of your calculator looks like the original design. 

Origal|Clone
:------:|:-------:
![image](https://user-images.githubusercontent.com/59168430/133880045-5e05ff34-a96d-4e29-a12e-cc3dbe53ca35.png)|![image](https://user-images.githubusercontent.com/59168430/133880054-07e21347-d323-4e6c-95ff-6becf0692ab8.png)